hs_app_netCDF/models.py

```diff
@@ -349,4 +263,10 @@ def add_metadata_element_to_xml(self, root, md_element, md_fields):
                                                                       field=md_fields[md_field]))
                     field.text = str(attr)
 
+    def delete_all_elements(self):
+        super(NetcdfMetaData, self).delete_all_elements()
+        if self.ori_coverage.all().first():
```

Comment:
This is just a single metadata element. I saw that for time series, the "site" is kind of similar for ori_coverage. Do I need to follow that one or remove the if statement?
if self.site:
            self.site.delete()